docs/installation/installing-octopus/automating-installation.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Automating Octopus installation
+description: Information on how to install and configure an Octopus server in a fully automated way from the command line.
+---
+
+## Octopus installers
+
+Octopus comes in a MSI that can be deployed via group policy or other means.
+
+The latest Octopus MSI can always be [downloaded from the Octopus Deploy downloads page](https://octopus.com/downloads).
+
+Permalinks to always get the latest MSIs are:
+
+- 32-bit: [https://octopus.com/downloads/latest/WindowsX86/OctopusServer](https://octopus.com/downloads/latest/WindowsX86/OctopusServer)
+- 64-bit: [https://octopus.com/downloads/latest/WindowsX64/OctopusServer](https://octopus.com/downloads/latest/WindowsX64/OctopusServer)
+
+Automating the installation of Octopus Server is a three step process.
+
+### 1. Install MSI on a temporary machine interactively
+In this step we install the MSI on a machine interactively so that we can complete the wizard to add a new instance.
+Follow all the steps in the wizard and once you get to the last screen, click on `Show Script` (**do not click Install**).
+![](show-script.png "width=500")
+
+Copy the script generated onto the clipboard.
+
+Here is an example what the script looks like, **you need to modify this script manually with your settings**:
+```bash
+"C:\Program Files\Octopus Deploy\MyOctopus\Octopus.Server.exe" create-instance --instance "new instance" --config "C:\Octopus\new instance\OctopusServer-new instance.config"
```

Comment:
These commands should be for the default installation location and default instance.  Keep the --instance parameter but make it use the default (I forget what it is off the top of my head)